crates/nu-cli/tests/completions/mod.rs

```diff
@@ -783,6 +839,59 @@ fn command_watch_with_filecompletion() {
     match_suggestions(&expected_paths, &suggestions)
 }
 
+#[test]
+fn command_stat_with_dot_expansion_completion() {
```

Comment:
I think you can remove this test, it was just me being silly.

Note for the future, though: `stat` is available on some systems but not others, so it'd be better to use a name like `nonexistent-command`, which definitely won't exist on any system. You very likely realized this already and I know you used `stat` specifically because I mentioned it, but I wanted to point it out just in case.

Also, it would've been easier to make the partial string look like `stat /foo/bar/...` rather than setting `$env.PWD`, but I guess you also did that specifically because I was using `stat .../`.